Packages/vcs/Lib/VTKPlots.py

```diff
@@ -1406,34 +1406,38 @@ def put_img_on_canvas(self,filename,zoom=1,xOffset=0,yOffset=0,*args,**kargs):
     cam.SetFocalPoint(xc+xoff,yc+yoff,0.)
     cam.SetPosition(xc+xoff,yc+yoff,d)
     ren.AddActor(a)
+    layer = self.renWin.GetNumberOfLayers() - 2
```

Comment:
Is it possible for layer to be less than 0 in this case? Might want to add a check if so.